src/core_plugins/kibana/common/field_formats/field_formats.js

```diff
@@ -0,0 +1,45 @@
+export class FieldFormatsService {
+  constructor() {
+    this.fieldFormats = new Map();
```

Comment:
This isn't actually a rule in our style guide, right?  Airbnb also says not to use the leading underscore (https://github.com/airbnb/javascript#naming--leading-underscore), though we never officially committed to airbnb style guide asfaik.

Maybe we need a "vote" issue for this so we can put something in our style guide, because I see inconsistent use of leading underscores.